src/org/sosy_lab/java_smt/api/FloatingPointFormulaManager.java

```diff
@@ -467,4 +529,13 @@ FloatingPointFormula multiply(
    * </ul>
    */
   BooleanFormula isNegative(FloatingPointFormula number);
+
+  /**
+   * Returns the size of the mantissa (also called a coefficient or significant), including the sign
+   * bit, for the given {@link FloatingPointFormula}.
+   */
+  int getMantissaSizeWithSignBit(FloatingPointFormula number);
+
+  /** Returns the size of the exponent for the given {@link FloatingPointFormula}. */
+  int getExponentSize(FloatingPointFormula number);
```

Comment:
Do we really need these two methods?
The user already can access the FormulaType directly and extract exponentSize and mantissaSize from the returned FP-FormulaType.